### DISCUSSION TITLE
Version 11 Figma Components

### DISCUSSION THREAD

**Discussion by jonmc12** [Shopify/polaris-react#9368] [Q&A] [answered]
Are the alpha version of the Figma components available for version 11? The [Version 11 Layout documentation](https://polaris.shopify.com/whats-new/version-11-layout) indicates they are available, but it does not provide a link.

**Comment by heyjoethomas** [comment, marked as answer]
Hey @jonmc12 the v11 Figma components are no longer available. Everything has been updated to reflect Polaris v12.
